galaxy_ng/app/api/v1/tasks.py

```diff
@@ -261,6 +262,7 @@ def legacy_sync_from_upstream(
         rkey = (ruser, rname)
         remote_id = rdata['id']
         role_versions = rversions[:]
+        github_user = rdata['github_user']
         github_repo = rdata['github_repo']
```

Comment:
`rdata['github_user']` is already read above into variable `ruser`. Then again, `ruser` is used instead of this variable to construct `clone_url`. Maybe this could be consolidated?